tests/Persist/BitRedisTest.php

```diff
@@ -121,7 +121,7 @@ public function setBits()
 
         $pipeMock->expects($this->once())
             ->method('exec')
-            ->willReturn(1);
+            ->willReturn(false);
```

Comment:
Just curious: did this test fail before all the time? Because 1 shouldn't be falsy 😉

Reply:
Tests were broken before, they were not starting because mock were inconsistent with library it is mocking documentation (there is no possibility of return integer here, according to phpdoc). Looks like php redis extension need to be updated on our side.

Reply:
Well that's rough. Thanks for fixing it!